### ISSUE TITLE
Setup.ps1 doesn't create build(.ps1|.sh|.cmd) files

### ISSUE THREAD

**Issue by hayhay27** [nuke-build/nuke#70]
In command line shell works perfectly

Steps to reproduce in powershell console:
```powershell
iwr https://nuke.build/powershell -OutFile setup.ps1
./setup.ps1
```

after setup build scripts doesn't exists

Fixed by
```powershell
#line 130
#WriteFile "build.ps1" ...
WriteFile "$RootDirectory/build.ps1" # ...

#line 137
#WriteFile "build.sh" ...
WriteFile "$RootDirectory/build.sh" # ...

#line 144
#(New-Object System.Net.WebClient).DownloadFile("$BootstrappingUrl/../build.cmd", "build.cmd")
(New-Object System.Net.WebClient).DownloadFile("$BootstrappingUrl/../build.cmd", "$RootDirectory/build.cmd")
```

----
```
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14409.1012
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14409.1012
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by arodus** [comment]
In this case `$RootDirectory` is not the directorty we are looking for because it's intended that the build scripts are written to the directory where the setup script was invoked. We fixed the bug by using `pwd`/`Get-Location` instead. 
Thanks for reporting this issue 👏

**Comment by matkoch** [comment]
Fixed in 0.5.0.
